tests/test_CtfEstimate.py

```diff
@@ -2,127 +2,115 @@
+    "defocus_v": 33944.32,  # angstrom
+    "defocus_ang": -65.26,  # Degree wrt some axis
+    "cs": 2.0,
+    "voltage": 300.0,
+    "pixel_size": 1.77,  # EMPIAR 10017
+    "amplitude_contrast": 0.07,
+}
 
-class CtfEstimatorTestCase(TestCase):
-    def setUp(self):
-        self.test_input_fn = "sample.mrc"
-        # These values are from CTFFIND4
-        self.test_output = {
-            "defocus_u": 34914.63,  # angstrom
-            "defocus_v": 33944.32,  # angstrom
-            "defocus_ang": -65.26,  # Degree wrt some axis
-            "cs": 2.0,
-            "voltage": 300.0,
-            "pixel_size": 1.77,  # EMPIAR 10017
-            "amplitude_contrast": 0.07,
-        }
-
-    def tearDown(self):
-        pass
-
-    def testEstimateCTF(self):
-        with tempfile.TemporaryDirectory() as tmp_input_dir:
-            # Copy input file
-            copyfile(
-                os.path.join(DATA_DIR, self.test_input_fn),
-                os.path.join(tmp_input_dir, self.test_input_fn),
+
+def test_estimate_CTF():
+    with tempfile.TemporaryDirectory() as tmp_input_dir:
+        # Copy input file
+        copyfile(
+            os.path.join(DATA_DIR, TEST_INPUT_FN),
+            os.path.join(tmp_input_dir, TEST_INPUT_FN),
+        )
+
+        with tempfile.TemporaryDirectory() as tmp_output_dir:
+            # Returns results in output_dir
+            results = estimate_ctf(
+                data_folder=tmp_input_dir,
+                pixel_size=1.77,
+                cs=2.0,
+                amplitude_contrast=0.07,
+                voltage=300.0,
+                num_tapers=2,
+                psd_size=512,
+                g_min=30.0,
+                g_max=5.0,
+                output_dir=tmp_output_dir,
+                dtype=np.float64,
+                save_ctf_images=True,
+                save_noise_images=True,
             )
 
-            with tempfile.TemporaryDirectory() as tmp_output_dir:
-                # Returns results in output_dir
-                results = estimate_ctf(
-                    data_folder=tmp_input_dir,
-                    pixel_size=1.77,
-                    cs=2.0,
-                    amplitude_contrast=0.07,
-                    voltage=300.0,
-                    num_tapers=2,
-                    psd_size=512,
-                    g_min=30.0,
-                    g_max=5.0,
-                    output_dir=tmp_output_dir,
-                    dtype=np.float64,
-                    save_ctf_images=True,
-                    save_noise_images=True,
-                )
+            logger.debug(f"results: {results}")
 
-                logger.debug(f"results: {results}")
+            for result in results.values():
+                # The defocus values are set to be within 5% of CTFFIND4
 
-                for result in results.values():
-                    # The defocus values are set to be within 5% of CTFFIND4
+                # defocusU
+                assert np.allclose(
```

Comment:
Since we're rewriting these, lets upgrade the `assert np.allclose` to `np.testing.assert_allclose` in this file.

This will print out a much nicer message when failing.

Reply:
echo tests/test_CtfEstimate.py | xargs -I% sed -i.bak "s/assert\ np.allclose/np.testing.assert_allclose/g" %

:)